Executive.js

```diff
@@ -1,41 +1,57 @@
 class Executive {
-	constructor(rows, cols) {
+	constructor(rows, cols, numbShip) {
 		this.rows = rows;
 		this.cols = cols;
 
 		// Which player's turn it is (players are false and true, aka 0 and 1)
-		this.turn = false;
-
-		this.board0 = new Board(rows, cols);
-		this.board1 = new Board(rows, cols);
-
-		this.renderBoards();
+		this.turn = false; 
+		this.numbShips = numbShip;
+		this.board0 = new Board(rows, cols, this.numbShips);
+		this.board1 = new Board(rows, cols, this.numbShips);
+		
+		this.renderBoards(false);
 
 		document.getElementById("switch-turn").addEventListener("click", e => {
-			this.turn = !this.turn;
-			this.renderBoards();
-		});
-
-		document.getElementById("ship-slider").addEventListener("input", e => {
-			document.getElementById("num-ships").innerHTML = e.target.value;
+			this.blankBoards();
+			let modal = document.getElementById("modal");
+			modal.style.display = "block"
+			let time = 5;
+			let timer  = setInterval(() => {
```

Comment:
I feel it would be better to just have an "I'm here" button instead of a timer, so nobody is waiting or risking seeing the opponent's board.

Reply:
The idea I had of a timer is that if the players want they can say how much time to wait in between turns so as you said they don't see their opponent's boards since blankBoards is called and until the time runs out no one can see any ships in either board